src/DurableTask.SqlServer/ManagedIdentitySettings.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using Newtonsoft.Json;
+
+namespace DurableTask.SqlServer
+{
+    /// <summary>
+    /// Configuration options for Managed Identity.
+    /// </summary>
+    public class ManagedIdentitySettings
+    {
+        public const string Resource = "https://database.windows.net/";
```

Comment:
In my testing, I noticed that I needed to add ".default" to the end of this URL before I could successfully fetch a token from Azure AD. I think this might be a newer requirement in the more modern versions of Azure.Identity.
```suggestion
        public const string Resource = "https://database.windows.net/.default";
```